src/Service/Symfony/RefreshOAuthTokenListener.php

```diff
@@ -38,15 +38,16 @@ public function onKernelException(ExceptionEvent $event): void
         }
 
         $event->setResponse(new RedirectResponse($this->router->generate('refresh_oauth_token', ['type' => $exception->type()])));
+        $event->stopPropagation();
```

Comment:
`setResponse` already stops propagation :scream:

Reply:
:thinking: actually yes, this was added with momentum, but the prudent always insured :surfer: